### DISCUSSION TITLE
Facial Recognition / CLIP Smart Search models comparison / benchmarks

### DISCUSSION THREAD

**Discussion by Rundervinkje** [immich-app/immich#14958] [Q&A] [answered]
Hello there,
I've been searching for an hour now (general internet, this page, discord) and thought i'd just ask for help now.
I was wondering if there is an overview/comparison of the available models and their intended use-case, accuracy, resource intensity and other possible metrics that would be usefull in deciding the right model for my case. (raspi 5, headless with docker, Dutch)

Thank you in advance for your thoughts / help

**Comment by mertalev** [comment]
The small and large variants are benchmarked [here](https://github.com/deepinsight/insightface/blob/master/model_zoo/README.md) if it helps. My recommendation is to just stick to the default unless you can’t run it.

**Comment by Rundervinkje** [comment]
@mertalev thank you for the quick response. Do you know if there is a similar page for the smart detection models? Then i'll rename this discussion post for future answer-lookers? 
I've been looking through the https://github.com/huggingface/transformers/tree/main but i honestly don't know what terms to search for.

**Reply by mertalev** [reply, marked as answer]
There's a chart [here](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_retrieval_results.csv) ([multilingual version](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_multilingual_retrieval_results.csv)) that goes into detail. It isn't that easy to read, though.

The guide [here](https://github.com/immich-app/immich/discussions/11862) has a chart I made based on this data and profiling. The models there are the ones I think are candidates to be the best (for English) within their size category.

**Reply by Rundervinkje** [reply]
You're a godsend! I'll read trough those. Happy holidays!
Edit: haha I see a familiar name on these posts!
